cmd/webhook.go

```diff
@@ -167,6 +185,26 @@ func waitForCerts(c crds.CertInfo, timeout time.Duration) error {
 	}
 }
 
+func getTLSCipherSuitesIDs(cipherListString string) ([]uint16, error) {
+	if cipherListString == "" {
+		return []uint16{}, nil
```

Comment:
the TLS docs say: `If CipherSuites is nil, a safe default list is used.` I believe we should return nil here, otherwise (if i understand the docs correctly) no ciphers are available for the client 🤔 🤷

Reply:
You are correct. Sorry about that. Fixed